.travis.yml

```diff
@@ -203,6 +203,13 @@ jobs:
       script:
         - travis_retry ./scripts/if_changed.sh ./scripts/pod_lib_lint.rb FirebaseStorage.podspec --skip-tests --platforms=ios
 
+    - stage: test
```

Comment:
This should be a change at line 32, not an addition.

Reply:
Wasn't really sure what I was doing there lol. make more sense now though- thank you! done